iree/compiler/Dialect/Flow/IR/FlowOps.cpp

```diff
@@ -263,6 +265,118 @@ static LogicalResult verifyVariableStoreIndirectOp(
 // flow.dispatch.region
 //===----------------------------------------------------------------------===//
 
+llvm::Optional<std::pair<DispatchRegionOp, Operation *>>
+DispatchRegionOp::formFromAnchorOp(Value workload, Operation *anchorOp,
+                                   OpBuilder &builder) {
+  builder.setInsertionPoint(anchorOp);
+  auto loc = anchorOp->getLoc();
+  // Map anchor into new dispatch region.
+  llvm::SmallVector<Value, 4> capturedInputs(anchorOp->getOperands());
+  llvm::SmallVector<Type, 1> types(anchorOp->getResultTypes().begin(),
+                                   anchorOp->getResultTypes().end());
```

Comment:
`llvm::to_vector<1>(anchorOp->getResultTypes())` will do this too (and then it sits better in the drOp create args and doesn't have to be a local, which makes it clearer these types are not being reused in the following code)